docs/drivers.md

```diff
@@ -22,6 +22,15 @@ To install the KVM2 driver, first install and configure the prereqs:
 * Ubuntu 16.x or older: `sudo apt install libvirt-bin libvirt-daemon-system qemu-kvm`
 * Fedora/CentOS/RHEL: `sudo yum install libvirt-daemon-kvm qemu-kvm`
 
+```
+Enable,start, and verify the libvirtd service has started. 
+```shell
+systemctl enable libvirtd.service
+systemctl start libvirtd.service
+systemctl status libvirtd.service
```

Comment:
Status seems a bit paranoid ?

Reply:
yeah just a little.  I think it may be good to check before moving to the next step.